tedana/workflows/tedana.py

```diff
@@ -482,7 +543,7 @@ def tedana_workflow(
     if isinstance(data, str):
         data = [data]
 
-    LGR.info("Loading input data: {}".format([f for f in data]))
+    LGR.info(f"Loading input data: {[f for f in data]}")
```

Comment:
Many of these may fall into the "defer code formatting changes to a PR dedicated to those formatting changes; keep this PR to strictly the addition of RobustICA" advice category.

Reply:
Agree. But this was a part of tedana code and I did not change it. But it is good that you picked it up and updated it.